Project21/Project21/Assignment 3.c

```diff
@@ -0,0 +1,46 @@
+////// ------------------------------------------------------------------------------------------------
+//// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+//// Copyright (c) Metamation India.
+//// ------------------------------------------------------------------
+//// Assignment 3
+//// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+//// ------------------------------------------------------------------------------------------------
+#include<stdio.h>
+#define INT_SIZE sizeof(int) * 8
+#define HEX_SIZE 8
+
+int DecimalToBinary () {// Function to convert a decimal number to binary
+   int n = 0, index = 0;// Declare variables: 'n' to store the input number
+   int binary[INT_SIZE]; // Array to store the binary representation of the number
+   printf ("Enter decimal number:");

```

Comment:
Move these lines printf ("Enter decimal number:"); int m = scanf ("%d", &n); to main function and pass the input to the DecimalToBinary() and DecimalToHexdecimal() as well.